### PR TITLE
Added Management URL for Mac & Windows Configuration Command

### PR DESCRIPTION
**Pull request by ashutoshpw**
Changes 
- Added Management URL Command to Dashboard for Windows and Mac [Similar to Ubuntu Command]
- Fixed few typos 
- Added Gitpod.yml for Quick Development Environment
- Improved the Dev Flow by adding a .env file and probably not breaking anything.

### CODE REVIEW

**Comment by braginini** [comment]
Hey @ashutoshpw 
Thank you for contributing! Great job!

**Comment by braginini** [comment]
@ashutoshpw 
Could you please split things into separate PRs? 
E.g. gitpod in unrelated to the AddPeer views.

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+tasks:
+  - init: npm install && npm run build
+    command: npm run start
+
+ports:
+  - port: 3000
+    visibility: private
+
+## If you are using Gitpod, you will need to configure the full endpoint on "Allowed Callback URLs"
+## in Auth0 -> Application -> Settings
```
Comment (braginini) [inline .gitpod.yml:12]: I think that gitpod integration should come as a separate PR

File: example.env [hunk 2]
```diff
@@ -0,0 +1,4 @@
+NETBIRD_MGMT_PORT=33071
+AUTH_AUDIENCE="https://***********.auth0.com/api/v2/" 
+AUTH_AUTHORITY=https://***********.auth0.com
+AUTH_CLIENT_ID=*****************************
```
Comment (braginini) [inline example.env:4]: Same as with gitpod.
I think that the dev flow improvement should come separately from the UI Changes

**Review by braginini — CHANGES_REQUESTED** [review]

**Comment by ashutoshpw** [comment]
Sure, will do that.

**Comment by mlsmaycon** [comment]
Hello @ashutoshpw, in the next couple of weeks we will be pushing some changes to the add-peer page. 

Let us know how you feel about the requested changes, we would love to get your contribution

**Pull request closed without merging**